### PR TITLE
grammar task les1 has been done

### PR DESCRIPTION
**Pull request by andrpech**

### CODE REVIEW

File: homework/english/andrii_pechyborshch/lecture1/grammar.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
+# Упражнения Present Simple и Present Continuous
+
+## Раскройте скобки, употребляя глаголы в Present Continuous или в Present Simple.
+
+1. Your mum is working today?
+2. Why this boys are talking now?
+3. Where he lives?
```
Comment (olgaz-skiba) [inline homework/english/andrii_pechyborshch/lecture1/grammar.md:7]: Where does he live?

File: homework/english/andrii_pechyborshch/lecture1/grammar.md [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+4. Their mother cooks very well?
```
Comment (olgaz-skiba) [inline homework/english/andrii_pechyborshch/lecture1/grammar.md:8]: Interrogative sentence
Does their mother cook very well?

File: homework/english/andrii_pechyborshch/lecture1/grammar.md [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+5. Lili is not rich, she does not drive a Mercedes.
+6. Let´s go out. It is not raining now.
+7. I have to go now. It is geting dark
+8. A young man usually stands at the window.
+9. Ann usually makes doll dresses herself.
+10. His wife is having coffee at the moment?
+11. Let’s go for a walk. It is not raining.
+12. Look! The boys are coming home.
+13. Maila likes Chinese food?
```
Comment (olgaz-skiba) [inline homework/english/andrii_pechyborshch/lecture1/grammar.md:17]: Interrogative sentence
Does Maila like Chinese food?

**Review by olgaz-skiba — DISMISSED** [review]
Igfm

**Review by olgaz-skiba — APPROVED** [review]

**Review by Denis-Pikus — APPROVED** [review]
lgfm

**Review by goldenas18 — APPROVED** [review]

**Pull request merged**